### ISSUE TITLE
Backward pass error when training with --Gnorm 'none'

### ISSUE THREAD

**Issue by BounharAbdelaziz** [chaofengc/Face-SPARNet#12]
Hi,

I was trying to train SPARNetHD without the normalization but I get the following error :

![image](https://user-images.githubusercontent.com/40790121/124910958-7dc0df00-dfec-11eb-8c11-e79652c265b7.png)

Even when setting `with torch.autograd.set_detect_anomaly(True) `I can't succesfuly run it :/

Thank you very much for your help!
With best,

**Comment by chaofengc** [comment]
This might because latest version of PyTorch is not compatible with the following identity lambda function:
https://github.com/chaofengc/Face-SPARNet/blob/5204283bcb584067a4b28e44231cf8f150cf07a3/models/blocks.py#L27

You may try to write an identity module to replace this lambda function.

**Comment by BounharAbdelaziz** [comment]
Thank you for your answear!

I have other architectures that uses the same code and that runs perfectly with the 'none' argument (thus the lambda function) :(

**Comment by chaofengc** [comment]
You may simply change the lambda function like this:
```
self.norm = lambda x: x * 1.0   
```
This helps to avoid inplace operation brought by newest version of pytorch.

**Comment by BounharAbdelaziz** [comment]
Thank you sir, this worked!
